ISHPullUp/ISHPullUpViewController.m

```diff
@@ -374,11 +381,12 @@ - (void)setBottomHeight:(CGFloat)bottomHeight animated:(BOOL)animated {
         return;
     }
 
-    [UIView animateWithDuration:0.4
+    ISHPullUpAnimationConfiguration config = self.animationConfiguration;
+    [UIView animateWithDuration:config.duration
                           delay:0
-         usingSpringWithDamping:0.9
-          initialSpringVelocity:0.3
-                        options:UIViewAnimationOptionAllowUserInteraction | UIViewAnimationOptionLayoutSubviews
+         usingSpringWithDamping:config.springDamping
+          initialSpringVelocity:config.initialVelocity
+                        options:config.options
                      animations:updateBlock
```

Comment:
we could also create an animation protocol delegate and call it from within this blog.